std/flash/_std/haxe/ds/ObjectMap.hx

```diff
@@ -60,8 +60,9 @@ class ObjectMap<K:{}, V> extends flash.utils.Dictionary implements haxe.Constrai
 
 	public function copy():ObjectMap<K, V> {
 		var copied = new ObjectMap();
+		@:nullSafety(Off)
 		for (key in keys())
-			copied.set(key, get(key));
+			copied.set(key, get(key)); @:nullSafety(Off)
```

Comment:
This is "unusual" formatting :)

Reply:
Yeah, formatter bug in current vshaxe version, i will run latest formatter from haxelib later...